src/app/migration/src/src/app/modules/workspace/components/content-creation-forms/create-collection/create-collection.component.ts

```diff
@@ -0,0 +1,136 @@
+import { UserService } from '@sunbird/core';
+import { FormControl, FormGroup, FormBuilder } from '@angular/forms';
+
+import { collectionDataInterface } from './../../../interfaces/collection.data.interface';
+import { SuiModule } from 'ng2-semantic-ui/dist';
+import { SuiModalService, TemplateModalConfig, ModalTemplate } from 'ng2-semantic-ui';
+import { Component, OnInit } from '@angular/core';
+import { ResourceService, ConfigService, ToasterService, ServerResponse, IUserData } from '@sunbird/shared';
+import { Router } from '@angular/router';
+import { EditorService } from './../../../services/editors/editor.service';
+
+
+@Component({
+  selector: 'app-create-collection',
+  templateUrl: './create-collection.component.html',
+  styleUrls: ['./create-collection.component.css']
+})
+export class CreateCollectionComponent implements OnInit {
+
+  isCollectionCreated: boolean;
+
+  /**
+	 * This variable hepls to show and hide page loader.
+   * It is kept true by default as at first when we comes
+   * to a page the loader should be displayed before showing
+   * any data
+	 */
+  showLoader = true;
+  /**
+  * To show toaster(error, success etc) after any API calls
+  */
+  private toasterService: ToasterService;
+  /**
+   * To call resource service which helps to use language constant
+   */
+  public userService: UserService;
+  /**
+    * To call resource service which helps to use language constant
+    */
+  public resourceService: ResourceService;
+  /**
+   * To make inbox API calls
+   */
+  public userForm: FormGroup;
+
+  private editorService: EditorService;
+  public state: string;
+  public type: string;
+  public framework: string;
+  public showEditor: boolean;
+  public contentId: any;
+  public requestBody: any;
```

Comment:
state, type, framework, contentId and  requestBody  should be a created as const inside menthod,  not as class member

Reply:
Done